scripts.d/support/install-setup-ng

```diff
@@ -197,10 +197,26 @@ $SUDO rm -f $prefix/setup
 
 $SUDO chown $USER -R $prefix
 
-# Setup the HPC servers, the template file servers.pvsc
-# will be used by the following script to create or update
-# the file $HOME/.config/ParaView
-./salome_hpc_visu_servers.py
+# Setup the HPC servers, the template file servers.pvsc will be used 
+# by the following script to replace (and copy with date if exist)
+# the file in $HOME/.config/ParaView
+
+local configPath="/home/$USER/.config/ParaView"
```

Comment:
I'm sure you have warnings here as local can only be used inside a function. Here you are in the main part of the script. Can you double check?

Reply:
Also please use all lower case for variables (this is the current convention scripts). So:

```
configPath      -> CONFIG_PATH
userServersPvsc -> USER_SERVER_PVSC
```